modules/ROOT/pages/rtc/getting-started.adoc

```diff
@@ -88,4 +91,4 @@ tinymce.init({
 });
 ----
 
-For details on the available configuration options, see: link:{baseurl}/rtc/configuration/[Configuration Options].
+For details on the available configuration options, see: xref:rtc/configuration.adoc[Configuration Options].
```

Comment:
This xref doesn't resolve... I believe it should be this instead:
```suggestion
For details on the available configuration options, see: xref:rtc/configuration/index.adoc[Configuration Options].
```

Reply:
done